src/ApplicationCore/Entities/FraudProtectionApiModels/Response/RuleEvaluation.cs

```diff
@@ -11,5 +11,7 @@ public class RuleEvaluation
         public string Rule { get; set; }
 
         public IEnumerable<string> ClauseNames { get; set; }
+        
+        public IEnumerable<string> EnvironmentId { get; set; }
```

Comment:
How come it's IEnumerable, but the field name is singular "EnvironmentId" instead of "EnvironmentIds"

Reply:
Very sorry about this.